bpf/lib/nodeport.h

```diff
@@ -637,6 +637,7 @@ int tail_nodeport_nat_ipv6(struct __ctx_buff *ctx)
 		}
 	}
 #endif
+#define DISABLE_ICMP_SUPPORT
```

Comment:
Should have defined this before to include nat.h ... I should take some holidays I guess.